### PR TITLE
Deploy Sentinel dVPN and Webtop

### PR DESCRIPTION
**Pull request by Dimokus88**
Deployment of the Sentinel dVPN node on Akash.
It's not a validator node, it's a dVPN backend node with v2RAY technology support.

### CODE REVIEW

**Comment by 88plug** [comment]
Need to update the README.md in the source of the repo.

**Comment by Dimokus88** [comment]
Add webtop deploy

File: README.md [hunk 1]
```diff
@@ -211,12 +211,14 @@ Awesome DeFi apps you can deploy on Akash
...
 - [Quill editor](quill-editor)
 - [Uptime Kuma](uptime-kuma)
+- [Webtop](webtop)
 - [Zammad](zammad)
 - [Budibase](budibase)
 
 ### Network
 
 - [SoftEther VPN](softether-vpn)
+- [Sentinel dVPN node](Sentinel%20dVPN%20node)
```
Comment (boz) [inline README.md:221]: Please don't use spaces in directory/file names.  Use `sentinel-dvpn-node` instead.

File: webtop/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+`fedora-i3`	✅	i3 Fedora <br/>
+`arch-i3`	✅	i3 Arch <br/>
+`alpine-openbox`	✅	Openbox Alpine <br/>
+`ubuntu-openbox`	✅	Openbox Ubuntu <br/>
+`fedora-openbox`	✅	Openbox Fedora <br/>
+`arch-openbox`	✅	Openbox Arch <br/>
+`alpine-icewm`	✅	IceWM Alpine <br/>
+`ubuntu-icewm`	✅	IceWM Ubuntu <br/>
+`fedora-icewm`	✅	IceWM Fedora <br/>
+`arch-icewm`	✅	IceWM Arch <br/>
```
Comment (boz) [inline webtop/README.md:63]: can you make this a markdown table?

**Review by boz — CHANGES_REQUESTED** [review]
Looking great!  Thanks for the contribution.  Some changes requested.

File: README.md (hunk 1 above)
Comment (Dimokus88) [reply README.md]: I fixed

File: webtop/README.md (hunk 2 above)
Comment (Dimokus88) [reply webtop/README.md]: Sure, fixed now.

File: README.md (hunk 1 above)
Comment (Dimokus88) [reply README.md]: Fix it

**Review by andy108369 — APPROVED** [review]

**Pull request merged**